components/Portfolio/Trading/Position/index.tsx

```diff
@@ -176,7 +175,7 @@ const Position: React.FC<{
                 </TableBody>
             </Table>
             <div className="flex mt-8 justify-center">
-                <Button theme={largeButton} onClick={(e) => onClick(e)}>
+                <Button theme={largeButton} onClick={(e: any) => onClick(e)}>
```

Comment:
Do button themes still exist here

Reply:
they don't, there's a lot we need to update for the other portfolio tabs

Reply:
If they arent used we should remove them as we find them